File: 240603/소수 판별기/prime-number-discriminator.py
N = int(input())

def is_prime_number(num):
    for i in range(2, int(num ** 0.5) + 1):
        if num % i == 0:
            return False

    return True

if is_prime_number(N):
    print('P')
else:
    print('C')

File: 251028/소수 판별기/prime-number-discriminator.cpp
#include <iostream>
#include <cmath>
using namespace std;

bool is_prime_number(int num)
{
    for(int i = 2; i <= int(pow(num, 0.5)) + 1; i++)
    {
        if(num % i == 0)
            return false;
    }

    return true;
}

int main() 
{
    // Please write your code here.
    int n;
    cin >> n;

    if(is_prime_number(n))
        cout << "P";
    else
        cout << "C";

    return 0;
}